COPYRIGHT

```diff
@@ -1 +1,5 @@
-Copyright (C) 2015 The Nightscout Foundation, http://www.nightscoutfoundation.org
```

Comment:
LOL right - you've accidentally created a pull request for the developers. Please disregard the rest of the comments! Apologies for comment spam. :D